operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml

```diff
@@ -217,6 +217,19 @@ spec:
                     description: loadBalancer holds parameters for the load balancer.
                       Present only if type is LoadBalancerService.
                     properties:
+                      allowedSourceRanges:
+                        description: allowedSourceRanges specifies a list of IP address
+                          ranges to which access to the load balancer should be restricted.  Each
+                          range must be specified using CIDR notation (e.g., "10.0.0.0/8").  If
+                          this field is empty, all source addresses are allowed (which
+                          is equivalent to specifying "0.0.0.0/0").
+                        items:
+                          description: CIDR is an IP address range in CIDR notation
+                            (for example, "10.0.0.0/8" or "fd00::/8").
+                          format: cidr
+                          type: string
+                        nullable: true
```

Comment:
Consider `nullable: false`, `default: "0.0.0.0/0"`, and removing that part of the description which says that will happen if allowedSourceRanges isn't specified.

Reply:
Done.

Reply:
Reverting this in favor of https://github.com/openshift/enhancements/pull/1177#discussion_r930203005.